diff --git a/dist/index.mjs b/dist/index.mjs
index ef6aacd9dc6950eb4230ee72ee4e8f4768cd6075..8a80ddfbf6c795ca29c6d7cf9823b7484888b87f 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -20,7 +20,7 @@ import * as CheckboxPrimitive from "@radix-ui/react-checkbox";
 import * as SelectPrimitive from "@radix-ui/react-select";
 import { UAParser } from "ua-parser-js";
 import { useAuthQuery } from "better-auth/client";
-import { anonymousClient, apiKeyClient, emailOTPClient, genericOAuthClient, magicLinkClient, multiSessionClient, oneTapClient, organizationClient, twoFactorClient, usernameClient } from "better-auth/client/plugins";
+import { anonymousClient, emailOTPClient, genericOAuthClient, magicLinkClient, multiSessionClient, oneTapClient, organizationClient, twoFactorClient, usernameClient } from "better-auth/client/plugins";
 import { createAuthClient } from "better-auth/react";
 import { Drawer } from "vaul";
 import { OTPInput, OTPInputContext } from "input-otp";
@@ -8267,7 +8267,6 @@ function SecuritySettingsCards({ className, classNames, localization }) {
 	});
 }
 var authClient = createAuthClient({ plugins: [
-	apiKeyClient(),
 	multiSessionClient(),
 	passkeyClient(),
 	oneTapClient({ clientId: "" }),
